arlo_server/routes.py

```diff
@@ -360,15 +361,42 @@ def require_audit_admin_for_organization(organization_id: Optional[str]):
     )
 
 
+ELECTION_NEW_SCHEMA = {
+    "type": "object",
+    "properties": {
+        "auditName": {"type": "string"},
+        "organizationId": {"type": "string"},
+    },
+    "required": ["auditName"],
+}
+
+
 @app.route("/election/new", methods=["POST"])
 def election_new():
-    info = request.get_json()
+    election = request.get_json()
+    validate(election, ELECTION_NEW_SCHEMA)
 
-    organization_id = info.get("organizationId", None) if info else None
+    organization_id = election.get("organizationId", None)
     require_audit_admin_for_organization(organization_id)
 
-    election_id = create_election(organization_id=organization_id)
-    return jsonify(electionId=election_id)
+    election = Election(
+        id=str(uuid.uuid4()),
+        audit_name=election["auditName"],
+        organization_id=organization_id,
+    )
+    db.session.add(election)
+
+    try:
+        db.session.commit()
+    except IntegrityError as e:
+        if e.orig.diag.constraint_name == "election_audit_name_organization_id_key":
```

Comment:
depending on the constraint name could be a tad dangerous, but I guess with tests maybe it's okay?

We should at some point figure out how to use Postgres's `on conflict`.

Reply:
I've used `on conflict` before, but I pretty much thought it was just for `upsert` logic. Not sure how it would help here.

Open to ideas about a better thing to key off of then constraint name, but I couldn't think of anything!